frontend/app/src/components/StreamlitDialog/DeployDialog/DeployListElement.tsx

```diff
@@ -16,20 +16,27 @@
 
 import React, { ReactElement } from "react"
 
+import { Info } from "@emotion-icons/material-outlined"
+
 import Checkmark from "@streamlit/app/src/assets/svg/checkmark.svg"
 
 import { StyledElement } from "./styled-components"
 
 export interface IDeployListElementProps {
   children?: React.ReactNode
   extraSpacing?: boolean
+  infoIcon?: boolean
 }
 
 function DeployListElement(props: IDeployListElementProps): ReactElement {
-  const { children, extraSpacing } = props
+  const { children, extraSpacing, infoIcon } = props
   return (
     <StyledElement extraSpacing={extraSpacing}>
-      <img src={Checkmark} alt={"Checkmark"} />
+      {infoIcon ? (
+        <Info height={16} width={16} />
```

Comment:
I think here it would be good to use `import Icon from "@streamlit/lib/src/components/shared/Icon"` and pass the material icon as the `content` to it and set the size, e.g. `"lg"`.

Reply:
done